rfcs/0098-community-team.md

```diff
@@ -0,0 +1,194 @@
+
+**Transparency** — We recognize that the whole community has a stake in decisions. We strive not to be arbitrary, to disclose when important decisions are made, and to make sure there is clear justification.
+
+**Advocacy** — We don't just recite our values, we live by them. We don't simply ignore attacks on our values, we speak up and take action to defend them.
+
+## Contacting the community team
+[contact]: #contact
+
+The community team will be contactable via a public email address listed on the NixOS website. This email address will forward to all community team members. Additionally, all community team members will list some form of individual contact information so community members can reach out to them directly in the case of an issue concerning other community team members or matters they are not comfortable disclosing to the entire community team.
+
+Emails sent to the community team are not to be shared outside of the community team, or to more people than need to be involved in the case of messages sent to individual team members.
+
+There will be a method to quickly notify all community team members in official public communication channels, in case of something needing urgent attention.
+
+Any of these contact methods may be used for disputes, for discussion of issues facing the community or individual members, as a way to raise safety concerns, and for any other reason that feels necessary.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+This effort will be a focus of controversy, especially at the outset. It is the position of these authors that it's a necessary controversy, and that things will get less intense once they settle.
+
+# Alternatives
+[alternatives]: #alternatives
+
+Failing to actively manage and lead the community makes it difficult for the community to practice any sort of values, not even seemingly "easy" values such as inclusion of minorities and an opposition to fascism. Every technical project is, necessarily, also a community, and it is important to live up to that responsibility for the betterment of society as a whole.
+
+Creating a hierarchical power structure based on specific individuals having semi-permanent roles is an approach that could initially be successful at reducing conflict. However, it would require a great deal of trust which may not exist, and it is not sustainable to indefinitely place that level of trust in specific people. By having a rotating team which includes *everyone*, we spread the necessary skills throughout the community and foster a culture which handles conflict in appropriate ways from the beginning.
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+* Will the community agree to go in this direction?
+
+* How can we work this into the technical framework of the Nix community? (discourse, mjolnir/matrix, github teams, etc.)
+
```

Comment:
How will abuse by the community team be handled?
We can not expect the community team to always make sound decisions, and in case they e.g. ban someone out of spite, unrelated to NixOS or them breaking any rules (for example due to a personal issue between the two), what can the banned do to become unbanned, without going through the community team?
What will the other community members do, when their fellow member goes rogue?
Community members are encouraged to form social bonds with each other, but won't this be problematic since they might not condemn fellow rogue community members?